src/client/OpenFin/provider.ts

```diff
@@ -1,29 +1,217 @@
 import OpenFin from "@openfin/core"
+import { ENVIRONMENT } from "client/constants"
+
+type ExternalClientMap = Map<
+  OpenFin.ApplicationIdentity["uuid"],
+  OpenFin.InteropClient
+>
+
+export interface ExternalContext extends OpenFin.Context {
+  _clientInfo?: {
+    uuid: string
+  }
+}
+
+function interopOverride(
+  InteropBroker: OpenFin.Constructor<OpenFin.InteropBroker>,
+): OpenFin.InteropBroker {
+  class Override extends InteropBroker {
+    public analyticsUuid: string
+
+    public limitCheckerUuid: string
+
+    public analyticsClients: ExternalClientMap
+
+    public limitCheckerClients: ExternalClientMap
+
+    constructor() {
+      super()
+      this.analyticsUuid = `reactive-analytics-${ENVIRONMENT}`
+      this.limitCheckerUuid = `reactive-trader-limit-checker` // TODO make env specific
```

Comment:
I'm not sure that the limit checker UUID is env specific?

Reply:
it should be

Reply:
Done